src/components/PatientCreationPage/CompletePatientButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import AddIcon from "@material-ui/icons/Add";
+import { makeStyles } from "@material-ui/core/styles";
+import { NavLink } from "react-router-dom";
+
+const CompletePatientButton = ({
+  handleClick,
+  disableButton,
+}: {
+  handleClick: () => any | null;
+  disableButton: boolean;
+}) => {
+  const classes = useCompletePatientButtonStyles();
+  return (
+    <Button
+      component={NavLink}
+      to="/"
+      onClick={handleClick}
+      variant="contained"
+      color="secondary"
+      classes={{
+        root: classes.root,
+      }}
+      disabled={disableButton}
+    >
+      Complete
+    </Button>
+  );
+};
+
+const useCompletePatientButtonStyles = makeStyles({
```

Comment:
Throughout the the codebase we have some files with styles defined before the component, and some with styles defined after. We should probably decide whether we want it before or after for consistency